### PR TITLE
Added BluetoothUtils to global scope

### PR DESCRIPTION
**Pull request by vinchan**

### CODE REVIEW

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/plugin.xml [hunk 1]
```diff
@@ -5,59 +5,60 @@
...
     
-    <js-module name="CommBluetooth" src ="www/CommBluetooth.js">
-    </js-module>
+   <js-module name="CommBluetooth" src ="www/CommBluetooth.js">
+   </js-module>
     
-    <js-module name="BluetoothUtils" src ="www/BluetoothUtils.js">
-    </js-module>
+   <js-module name="BluetoothUtils" src ="www/BluetoothUtils.js">
+      <clobbers target="bluetoothUtils" />
```
Comment (kaysabelle) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/plugin.xml:35]: THE ONLY CHANGE IS HERE. nice

**Comment by MinickLS** [comment]
Cool.

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/plugin.xml (hunk 1 above)
Comment (jeslin) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/plugin.xml:35]: Good spacing fixes yay! :100:

**Comment by kaysabelle** [comment]
Looks good to me. I sign off.

**Comment by kaysabelle** [comment]
:+1:

**Pull request merged**